pages/community/contributors/index.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import GenericLayout from '../../../components/layout/GenericLayout';
+import Filter from '../../../components/navigation/Filter';
+import Heading from '../../../components/typography/Heading';
+import ambassadors from '../../../config/AMBASSADORS_MEMBERS.json';
+
+function Index() {
+  const image = '/img/social/website-card.png';
+  const [posts, setPosts] = useState(ambassadors);
+  return (
+    <GenericLayout
+      title="AsyncAPI Ambassador Program"
+      description="The home for developer communities"
```

Comment:
```suggestion
      description="Mentorship and OSS Community"
```

I don't like only focusing on saying this is all for "developers." Developers are not the only part of our OSS project, there are many kinds of contributions and contributors.